mockwebserver/src/main/java/okhttp3/mockwebserver/QueueDispatcher.kt

```diff
@@ -81,8 +81,10 @@ open class QueueDispatcher : Dispatcher() {
      * Enqueued on shutdown to release threads waiting on [dispatch]. Note that this response
      * isn't transmitted because the connection is closed before this response is returned.
      */
-    private val DEAD_LETTER = MockResponse()
-        .setStatus("HTTP/1.1 $HTTP_UNAVAILABLE shutting down")
+    private val DEAD_LETTER = run {
+      val mockResponse = MockResponse()
+      mockResponse.apply { mockResponse.status = "HTTP/1.1 $HTTP_UNAVAILABLE shutting down" }
+    }
```

Comment:
InteliJ translated the deprecated fun to the code above.
Maybe it's better to do like this?
`private val DEAD_LETTER = MockResponse().apply { this.status = "HTTP/1.1 $HTTP_UNAVAILABLE shutting down" }`

Reply:
Yes!